search-endpoint.md

```diff
@@ -4,47 +4,58 @@
 ## Search Endpoint
 **/api/discover/search**   
 
-This endpoint provides the functionality of the Discovery search screen: http://demo.dspace.org/xmlui/discover.
-
-Provide access to the Discovery search system (SOLR based). This endpoint returns the list of available DSpace object (DSO) types that can be searched:
-* all: Execute a query over all available DSO types.
-* items: Only search the DSpace items.
-* communities: Search within the DSpace community records.
-* collections: Limit the search to DSpace collection records.
-
-Example: TODO
-
-### Search with a given DSO type
-**/api/discover/search/<:dso-type>**
-
-This endpoint will provide detailed information on the Discovery search configuration that can be used to build complex searches.
+This endpoint provides the functionality of the Discovery search screen: https://wiki.duraspace.org/display/DSDOC6x/Discovery.
+It will provide detailed information on the Discovery search configuration that can be used to build complex searches.
 
 It supports the following parameters:
 * `scope`: UUID of a specific DSpace container (site, community or collection) to which the search will to be limited. If the scope has a specific DSpace configuration defined in `config/spring/api/discovery.xml`[https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/discovery.xml#L28] that configuration will be returned. Otherwise the default configuration will be returned.
+* `configuration`: The name of a Discovery configuration that should be used by this search. If the provided scope already has a specific Discovery configuration defined, than this parameter will be ignored.
```

Comment:
instead to ignore it I suggest to use it to override the default configuration. The configuration parameter should be optional and a default is applied